package.json

```diff
@@ -23,9 +23,9 @@
   },
   "homepage": "https://github.com/DistributedProofreaders/dproofreaders",
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "geckodriver": "^2.0.4",
-    "lessc": "^1.0.2",
-    "selenium-webdriver": "^4.0.0-rc-1"
+    "eslint": "^8.19.0",
+    "geckodriver": "^3.0.2",
+    "less": "^4.1.3",
```

Comment:
Has the script been renamed to `less` instead of `lessc`?

Reply:
No, the script is still called lessc. It seems the lessc package in npmjs is really old, unmaintained, with low download activity:
https://www.npmjs.com/package/lessc
Where if you read the docs on less's website:
https://lesscss.org/usage/
it says to use the less package which is maintained and has tons of download activity:
https://www.npmjs.com/package/less

I was curious so I looked at the package contents of the lessc artifact (since there is no linked source in npmjs) and it is just a warpper around an older version of less:
```
{
  "name": "lessc",
  "version": "1.0.2",
  "description": "Run lessc via npx",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "techknowlogick",
  "license": "ISC",
  "dependencies": {
    "less": "^3.9.0"
  }
}
```
So really my change just removes an unused middleman and updates the version to the latest. I am curious why we ever went with lessc at all. Perhaps it was recommended at some point? @cpeel seems to have implemented it with https://github.com/DistributedProofreaders/dproofreaders/commit/a8f179bd8d23954fd27f8b994cc5512ee6c02444.

Reply:
:D, I believe you nodejs knowledge is probably 10x my PHP knowledge, but we all try and that's what matters.